stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[AnyStr]]: ...
```

Comment:
I think it's `IO[str]` (https://github.com/python/cpython/blob/2.7/Lib/SimpleHTTPServer.py#L91).